.github/ISSUE_TEMPLATE/docs.yaml

```diff
@@ -32,7 +32,7 @@ body:
     attributes:
       label: Documentation location
       description: Location of the corresponding documentation page (e.g., file, section, URL), if applicable
-      placeholder: ex. https://canonical-rockcraft.readthedocs-hosted.com/en/latest/tutorials/hello-world/
+      placeholder: ex. https://canonical-rockcraft.readthedocs-hosted.com/latest/tutorial/hello-world/
```

Comment:
why don't you use the friendly ubuntu link here too?

Reply:
yep